maltoolbox/attackgraph/node.py

```diff
@@ -25,7 +25,7 @@ class AttackGraphNode:
     is_necessary: bool = True
     compromised_by: list[Attacker] = field(default_factory=list)
     mitre_info: Optional[str] = None
-    tags: list[str] = field(default_factory=lambda: [])
+    tags: set[str] = field(default_factory=lambda: {})
     attributes: Optional[dict] = None
```

Comment:
The `info` lines are missing from attack steps. I can see why someone would not like them to be part of an attack graph's nodes, but these are completely inaccessible currently from what it seems. There is the `extras` dict that can be used but it is not used (and it does not seem the best place as `extras` sounds like per-application-and-use-case custom info). The simplest solution is to add an `info` attribute that is set to the language graph's step description.

Reply:
Yes, this is one of the things I was having a look at when I was working on the latest commit. I think my version had some info strings in there, but some were still missing. Same thing with the tags, but I do not exactly recall what state it was left in.